tasks/main.yml

```diff
@@ -1,15 +1,14 @@
 ---
-# Set Rsyslog facts
+- name: Set Rsyslog facts then include rsyslog role
+  block:
 
-- block:
-
-    - name: Set rsyslog_enabled
+    - name: Set __rsyslog_enabled
       set_fact:
-        rsyslog_enabled: "{{ logging_enabled | d(true) }}"
+        __rsyslog_enabled: "{{ logging_enabled }}"
```

Comment:
if you remove the `| bool` filter on all places where `__rsyslog_enabled` is being used, you should add it here, as logging_enabled is user-supplied and not guaranteed to be a boolean - it can be a boolean-like string.

Reply:
@pcahyna that means we need to do input normalization on some of the input parameters?

Reply:
For booleans this seems to be the usual practice, yes, but I don't have a reference for this. Again, to be discussed.